### PR TITLE
Modify add-env help command examples

### PR DESCRIPTION
**Pull request by npamudika**

### CODE REVIEW

File: import-export-cli/cmd/addEnv.go [hunk 1]
```diff
@@ -43,23 +43,24 @@ const addEnvCmdExamples = utils.ProjectName + ` ` + addEnvCmdLiteral + ` -e prod
...
---publisher https://localhost:9443/api/am/publisher/v1/apis \
+--registration https://localhost:9443 \
+--publisher https://localhost:9443 \
 --devportal  https://localhost:9443 \
 --admin  https://localhost:9443 \
 --token https://localhost:8243/token
 
-NOTE: The flag --environment (-e) is mandatory
+NOTE: The flags --environment (-e) and --token are mandatory
 You can either provide only the 2 tags --apim and --token, or all the other 5 tags (--registration --publisher --devportal --admin --token) without providing --apim tag,
```
Comment (malinthaprasan) [inline import-export-cli/cmd/addEnv.go:61]: replace tags -> flags

File: import-export-cli/cmd/addEnv.go [hunk 2]
```diff
@@ -43,23 +43,24 @@ const addEnvCmdExamples = utils.ProjectName + ` ` + addEnvCmdLiteral + ` -e prod
...
-or all the 6 tags to add an environment.`
+or all the 6 tags to add an environment.
```
Comment (malinthaprasan) [inline import-export-cli/cmd/addEnv.go:62]: Remove this line.

File: import-export-cli/cmd/addEnv.go (hunk 1 above)
Comment (npamudika) [reply import-export-cli/cmd/addEnv.go]: done

File: import-export-cli/cmd/addEnv.go (hunk 2 above)
Comment (npamudika) [reply import-export-cli/cmd/addEnv.go]: done

File: import-export-cli/cmd/addEnv.go [hunk 3]
```diff
@@ -43,23 +43,23 @@ const addEnvCmdExamples = utils.ProjectName + ` ` + addEnvCmdLiteral + ` -e prod
...
-
-NOTE: The flag --environment (-e) is mandatory
-You can either provide only the 2 tags --apim and --token, or all the other 5 tags (--registration --publisher --devportal --admin --token) without providing --apim tag,
-or all the 6 tags to add an environment.`
+--apim https://apidev.com:9443 \
+--registration https://idp.com:9443 \
+--publisher https://apidev.com:9443 \
+--devportal  https://apps.com:9443 \
+--admin  https://apidev.com:9443 \
+--token https://gw.com:8243/token
```
Comment (malinthaprasan) [inline import-export-cli/cmd/addEnv.go:58]:
```suggestion
--apim https://apim.com:9443 \
--registration https://idp.com:9443 \
--token https://gw.com:8243/token
```

**Pull request merged**